### PR TITLE
feat: more work on query renderers exercise

### PR DESCRIPTION
**Pull request by pepopowitz**
Does two things with the query renderers exercise:

1. Makes some changes to the "starting point" for the query renderers exercise so that there's less noise/moving things around as the reader works through the exercise
2. Adds some more steps to the instructions.

### CODE REVIEW

File: src/exercises/01-Query-Renderer/README.md [hunk 1]
```diff
@@ -59,10 +83,96 @@ Let's update this page to show actual data about the current artist.
...
+💻 _Add an import statement for some dependencies to Artist1QueryRenderer:_
+
+```typescript
+import { graphql, QueryRenderer } from 'react-relay';
+import { environment } from '../../relay';
+```
+
+_TODO: line numbers_
+
+`QueryRenderer` is the component we're going to render; `graphql` helps us specify the GraphQL query that should be made.
```
Comment (xtina-starr) [inline src/exercises/01-Query-Renderer/README.md:107]: I wonder if it worth mentioning that the `QueryRenderer` is the root level of a query or relay tree. Calling out that in the Relay world it's the start of composing a query.

**Comment by xtina-starr** [comment]
This is looking good. I had one minor comment but if you don't feel strongly feel free to merge. Everything else looks great.

**Review by xtina-starr — APPROVED** [review]
🎉

File: src/exercises/01-Query-Renderer/README.md (hunk 1 above)
Comment (pepopowitz) [reply src/exercises/01-Query-Renderer/README.md]: Addressed in b37a8d2!

**Pull request merged**